src/elements.web.ts

```diff
@@ -262,8 +259,14 @@ export class Svg extends WebShape<BaseProps & SvgProps> {
 
     const rect = getBoundingClientRect(ref);
 
-    const width = Number(options.width) || rect.width;
-    const height = Number(options.height) || rect.height;
+    const width = Number(options?.size?.width) || rect.width;
+    const height = Number(options?.size?.height) || rect.height;
+    const format = options.format === 'jpeg' ? 'image/jpeg' : 'image/png';
+
+    let quality: number | undefined;
+    if (options && options.format === 'jpeg' && options.quality) {
```

Comment:
Is quality=0 a valid value? If yes, then we should check for `options.quality !== undefined` not for a truthy value.

Reply:
@jakex7 Don't we need to check the null variable?

Reply:
updated that check value.